docs/guides/serving.md

```diff
@@ -51,6 +51,8 @@ if __name__ == "__main__":
 
 You can also view a complete example [here](https://github.com/floydhub/fast-style-transfer/blob/master/app.py).
 
+!!! important "Pay attention to Race Conditions!"
+    The uswgi application server will spawn 2 workers (processes) for serving your application. This can be a source of race condition: remember to lock the resources every time you will need to write and read from them. 
```

Comment:
memory read write is ok, because each worker run in its own process. It's fairly safe to restrict it to disk resource for now.

Reply:
Ok, I've tried to be as generic as possible. I'll restrict it to disk resources then.